cmd/node/config/genesisSmartContracts.json

```diff
@@ -5,7 +5,7 @@
     "vm-type": "0500",
     "init-parameters": "%auction_sc_address%@03E8@03E8@03E8",
     "type": "delegation",
-    "version": "0.3.*"
+    "version": "0.4.*"
```

Comment:
:+1: 

Please replace the L87-L88 in /genesis/process/intermediate/baseDeploy.go with
```GO
			dp.pubkeyConv.Encode(scResultingAddressBytes),
			dp.pubkeyConv.Encode(ownerAddress),
```